.github/workflows/ci.yml

```
@@ -106,6 +106,25 @@ jobs:
               - 'cmd/**'
             gomod:
               - 'go.{mod,sum}'
+  fossa:
+    name: FOSSA License Scan
+    needs: detect-changes
+    if: github.event_name != 'merge_group' && secrets.FOSSA_API_KEY != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)